app/src/main/java/org/mozilla/reference/browser/browser/FirefoxAccountsIntegration.kt

```diff
@@ -110,7 +140,7 @@ class FirefoxAccountsIntegration(
     private fun persistProfile(profile: Profile) {
```

Comment:
(not your PR, but since I'm reading this code critically...) As far as I can tell, we never actually persist the profile. We persist account state (account state machine state, tokens, config, etc). The profile itself is cached in memory at the rust lib and at the kotlin level, and re-fetched every time app is re-launched (i think).

So this function is rather poorly named!

Reply:
OK, updated the naming here. This was just taken over from the origin FxA sample app.